<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>831</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>30</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_order">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>350</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>주문관리</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cs">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>350</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>문의관리</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_product">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>350</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>상품등록</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>306</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>비밀번호 찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_Button">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>420</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 15pt &quot;GyeonggiBatang&quot;;
	color: rgb(82, 194, 105);
	background-color: white;
	border: 2px solid  rgb(82, 194, 105);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>304</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>아이디 찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_Button">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>360</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 15pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(82, 194, 105);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>265</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>205</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;경기천년바탕 Regular&quot;;</string>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_6">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>306</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>|</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>115</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;경기천년바탕 Regular&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;로그인&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>511</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home1">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 13pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(82, 194, 105);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <zorder>signup_Button</zorder>
    <zorder>sh_label_4</zorder>
    <zorder>login_Button</zorder>
    <zorder>id</zorder>
    <zorder>pw</zorder>
    <zorder>label_7</zorder>
    <zorder>label_6</zorder>
    <zorder>sh_label_6</zorder>
    <zorder>sh_label</zorder>
    <zorder>line</zorder>
    <zorder>btn_home1</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>456</x>
       <y>41</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;GyeonggiBatang&quot;;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;회원가입&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>81</y>
       <width>571</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>181</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;아이디&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>231</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;비밀번호&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>309</x>
       <y>281</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;비밀번호 확인&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>334</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;연락처&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>381</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;주소&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>131</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;이름&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="joinid">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>171</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="joinpw">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>221</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="joinpw2">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>271</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>321</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>숫자만 입력하세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>371</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>111</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_duplication">
     <property name="geometry">
      <rect>
       <x>696</x>
       <y>173</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 11pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(82, 194, 105);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>중복확인</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>421</y>
       <width>571</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_join">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>460</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(82, 194, 105);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>가입하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>460</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: rgb(82, 194, 105);
	background-color: white;
	border: 2px solid rgb(82, 194, 105);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>96</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>상품등록</string>
     </property>
    </widget>
    <widget class="QComboBox" name="material">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>395</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>재료재고</string>
     </property>
    </widget>
    <widget class="QComboBox" name="material5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>430</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>430</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="productname">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>781</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>상품명</string>
     </property>
    </widget>
    <widget class="QComboBox" name="material2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="material3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>130</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock3">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="material4">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>130</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock4">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="material6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>430</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>430</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="material7">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>430</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock7">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>430</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="material8">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>430</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="stock8">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>430</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_check">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>740</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home3">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>주문관리</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="orderlist">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>111</y>
       <width>891</width>
       <height>321</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>고객</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문상품</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주문상태</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_check2">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>460</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home4">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>문의관리</string>
     </property>
    </widget>
    <widget class="QListWidget" name="customerservice">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>131</y>
       <width>901</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_check3">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>470</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home5">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
